documentation/book/assembly-kafka-bridge-quickstart.adoc

```diff
@@ -0,0 +1,57 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-using-the-kafka-bridge.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context-using-the-kafka-bridge: {context}
+:context: {parent-context-using-the-kafka-bridge}-bridge-quickstart
+
+[id='assembly-kafka-bridge-quickstart-{context}']
+= Kafka Bridge quick start
+
+Use this quick start to try out the {ProductName} Kafka Bridge in your local development environment. You will learn how to:
+
+* Deploy the Kafka Bridge to your OpenShift cluster
+* Expose the Kafka Bridge Service to your local machine
+* Produce messages to topics, including to a specific partition
+* Create a Kafka Bridge consumer for use in this quick start
+* Perform basic consumer operations
+
+HTTP requests are formatted as curl commands that you can copy and paste to your terminal. You can use an API testing tool to call the API if preferred. As described in xref:con-requests-kafka-bridge-str[Requests to the {ProductName} Kafka Bridge], all POST requests used in this quick start supply the following Content-Type header:
+
+[source,http,subs=attributes+]
+----
+Content-Type: application/vnd.kafka.v2+json
+----
+
+You must have access to a Kubernetes cluster. To run and manage a Kubernetes cluster locally, use a tool such as Minikube, CodeReady Containers, or MiniShift.
+
+.Prerequisites for the quick start
+
+* Cluster administrator access to a local or remote Kubernetes cluster. 
+* {ProductName} is installed.
+* A running Kafka cluster, deployed by the Cluster Operator, in a Kubernetes namespace.
+* The Cluster Operator `RoleBindings` are installed.
+* The Entity Operator is deployed and running.  
```

Comment:
We should try to avoid confusion that this needs to be deployed separately.

```suggestion
* The Entity Operator is deployed and running as part of the Kafka cluster.  
```